index.html

```diff
@@ -1,118 +1,124 @@
-                <li>
-                    <strong>1962</strong> - Visits Delhi and brings his high-yielding
-                    strains of wheat to the Indian subcontinent in time to help mitigate
-                    mass starvation due to a rapidly expanding population
-                </li>
-                <li><strong>1970</strong> - receives the Nobel Peace Prize</li>
-                <li>
-                    <strong>1983</strong> - helps seven African countries dramatically
-                    increase their maize and sorghum yields
-                </li>
-                <li>
-                    <strong>1984</strong> - becomes a distinguished professor at Texas
-                    A&M University
-                </li>
-                <li>
-                    <strong>2005</strong> - states "we will have to double the world
-                    food supply by 2050." Argues that genetically modified crops are the
-                    only way we can meet the demand, as we run out of arable land. Says
-                    that GM crops are not inherently dangerous because "we've been
-                    genetically modifying plants and animals for a long time. Long
-                    before we called it science, people were selecting the best breeds."
-                </li>
-                <li><strong>2009</strong> - dies at the age of 95.</li>
-            </ul>
-            <blockquote
-                cite="http://news.rediff.com/report/2009/sep/14/pm-pays-tribute-to-father-of-green-revolution-borlaug.htm">
-                <p>
-                    "Borlaug's life and achievement are testimony to the far-reaching
-                    contribution that one man's towering intellect, persistence and
-                    scientific vision can make to human peace and progress."
-                </p>
-                <cite>-- Indian Prime Minister Manmohan Singh</cite>
-            </blockquote>
-            <h3>
-                If you have time, you should read more about this incredible human
-                being on his
-                <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Wikipedia
-                    entry</a>.
-            </h3>
-        </section>
+      <h5 id="title">Dr. Norman Borlaug</h5>
+      <span>The man who saved a billion lives</span>
+      <figure id="img-div">
+        <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
+        />
+        <figcaption id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </figcaption>
+      </figure>
+      <section id="tribute-info">
+        <h1 id="headline">Here's a time line of Dr. Borlaug's life:</h1>
```

Comment:
It is best practice to use the HTML tags as per semantics. `h1` tag should be used for the main heading on the website.  I suggest using the `hx` tag instead of `h1`.